### PR TITLE
fix wait help info

### PR DESCRIPTION
**Pull request by qiaozha**

### CODE REVIEW

**Comment by qiaozha** [comment]
to fix issue https://github.com/Azure/autorest.az/issues/335

File: src/test/scenarios/managed-network/output/src/managed-network/azext_managed_network/generated/_help.py [hunk 1]
```diff
@@ -74,6 +75,18 @@
                az managed-network mn delete --name "myManagedNetwork" --resource-group "myResourceGroup"
 """
 
+helps['managed-network mn wait'] = """
+    type: command
+    short-summary: Place the CLI in a waiting state until a condition of the managed-network mn is met.
+    examples:
+      - name: Pause executing next line of CLI script until the managed-network mn is successfully provisioned.
+        text: |-
+               az managed-network mn wait --name "myManagedNetwork" --resource-group "myResourceGroup" updated
```
Comment (fengzhou-msft) [inline src/test/scenarios/managed-network/output/src/managed-network/azext_managed_network/generated/_help.py:84]: should be --updated

File: src/test/scenarios/managed-network/output/src/managed-network/azext_managed_network/generated/_help.py [hunk 2]
```diff
@@ -74,6 +75,18 @@
...
 """
 
+helps['managed-network mn wait'] = """
+    type: command
+    short-summary: Place the CLI in a waiting state until a condition of the managed-network mn is met.
+    examples:
+      - name: Pause executing next line of CLI script until the managed-network mn is successfully provisioned.
+        text: |-
+               az managed-network mn wait --name "myManagedNetwork" --resource-group "myResourceGroup" updated
+      - name: Pause executing next line of CLI script until the managed-network mn is successfully provisioned.
```
Comment (fengzhou-msft) [inline src/test/scenarios/managed-network/output/src/managed-network/azext_managed_network/generated/_help.py:85]: The example names are the same now. Can you make the them different?

File: src/test/scenarios/managed-network/output/src/managed-network/azext_managed_network/generated/_help.py (hunk 1 above)
Comment (qiaozha) [reply src/test/scenarios/managed-network/output/src/managed-network/azext_managed_network/generated/_help.py:84]: ohh right

File: src/test/scenarios/managed-network/output/src/managed-network/azext_managed_network/generated/_help.py (hunk 2 above)
Comment (qiaozha) [reply src/test/scenarios/managed-network/output/src/managed-network/azext_managed_network/generated/_help.py:85]: how?

Comment (fengzhou-msft) [reply src/test/scenarios/managed-network/output/src/managed-network/azext_managed_network/generated/_help.py:85]: change `provisioned` to `created/updated/deleted`? Add @haroldrandom to comment who is familiar with help text conventions.

Comment (qiaozha) [reply src/test/scenarios/managed-network/output/src/managed-network/azext_managed_network/generated/_help.py:85]: Maybe we can add the created or updated in the name?

File: src/test/scenarios/managed-network/output/src/managed-network/azext_managed_network/generated/_help.py (hunk 1 above)
Comment (qiaozha) [reply src/test/scenarios/managed-network/output/src/managed-network/azext_managed_network/generated/_help.py]: Changed

File: src/test/scenarios/managed-network/output/src/managed-network/azext_managed_network/generated/_help.py (hunk 2 above)
Comment (qiaozha) [reply src/test/scenarios/managed-network/output/src/managed-network/azext_managed_network/generated/_help.py]: I changed `provisioned` into `created/updated/deleted` @haroldrandom Can you confirm if this is okay for wait command?

Comment (haroldrandom) [reply src/test/scenarios/managed-network/output/src/managed-network/azext_managed_network/generated/_help.py]: what do `provisioned ` mean?

Comment (qiaozha) [reply src/test/scenarios/managed-network/output/src/managed-network/azext_managed_network/generated/_help.py]: provisioningStatus is Success?

**Pull request merged**